roles/nginx/handlers/main.yml

```diff
@@ -0,0 +1,12 @@
+---
+- name: restart nginx
+  become: yes
+  service:
+    name: nginx
+    state: restarted
+
+- name: reload nginx
```

Comment:
Not used right now but brings just a bit more complete role.
Still can be used in future and in our external infra where we just include `nginx` role and use it how we like it.
Ex: install nginx in internal infra, same for `mongo`, `postgres` or any other role from this repo.